cmd/weaver-kube/deploy.go

```diff
@@ -116,6 +116,10 @@ func deploy(ctx context.Context, args []string) error {
 	if config.Namespace == "" {
 		config.Namespace = "default"
 	}
+	if config.ServiceAccount == "" {
```

Comment:
If we leave `ServiceAccount` empty, does Kubernetes use the default one? Same for namespace?

Reply:
No, it doesn't work. I tried that before for namespace as well. It generates an empty string even if I don't explicitly set that field.